### DISCUSSION TITLE
How to use mdx-parallax and others to build a fancy landing page

### DISCUSSION THREAD

**Discussion by RealLast** [squidfunk/mkdocs-material#4262] [Q&A] [answered]
Hello everyone,

I just found this project and have to say I am very impressed! I was looking for a way to easily make pretty and intuitive documentations in a modern design. This seems to be exactly what I need.

So I was looking around a bit and trying everything, and most of it is very intuitive. Then I wanted to go ahead and build a simple landing page. I took some inspiration from your landing page on https://squidfunk.github.io/mkdocs-material/, because I think it is very fancy. I wanted to integrate some of the used blocks to my page aswell, thus I was inspecting the website to see how it was build.
I found that some of the used blocks are "mdx-parallax__group" and "mdx-spotlight__feature". But I did not find a way to integrate them into my website.. Is it possible to use those mdx features, e.g. the parallax effect, for custom landing pages?

Best regards

**Comment by squidfunk** [comment, marked as answer]
Thanks! It's definitely possible to use and adjust the custom templates. The source code is part of the [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/) private repository (not this one), primarily due to copyright reasons concerning the hero graphic.

Every class prefixed with `md-*` is native functionality, while `mdx-*` denotes custom extensions for our own documentation.

**Comment by RealLast** [comment]
Hi,

thank you very much for your response!
So, if I understand correctly, everything within the mdx-* namespace is only available from the Insiders private repository, is that correct? So if I decide to become an insider/sponsor, I could access and use that aswell?


Best

**Reply by squidfunk** [reply]
Some `mdx-*` declarations are also in the public repository, but all stuff that is related to the landing page is in Insiders, which you're granted access to when sponsoring, yes.

**Reply by RealLast** [reply]
Cool, thanks for clarification! I am going to write my documentation and if all goes well I am considering becoming insider - I really like this project ! 👍🏼

**Comment by snakers4** [comment]
Hi @squidfunk,

My friend uses MkDocs to publish her book, because apparently MkDocs works well as book "engine". She would like to build a nice landing page similar to your landing page (https://squidfunk.github.io/mkdocs-material/).

Is it possible to contract a custom landing page template for a book from you with a simple installation guide?

**Reply by squidfunk** [reply]
Thanks for asking! Unfortunately, I currently can't offer contract work, because I'm focusing on Material for MkDocs ☺️ However, [as mentioned before](https://github.com/squidfunk/mkdocs-material/discussions/4262#discussioncomment-3429880), you can just become a sponsor, and use [our official landing page template](https://github.com/squidfunk/mkdocs-material-insiders/tree/master/src/overrides) (The link only works when you sponsored). If you want nice graphics, I recommend getting in touch with [BewilderedBird](https://mastodon.art/@BewilderedBird), as she did the illustration on our landing page. She's definitely open to contract work and might be able to help you.

**Reply by snakers4** [reply]
Very nice, thank you! We will start with the graphics.

**Reply by md42** [reply]
@squidfunk do you know if there is still an online presence of BewilderedBird? I like the landing page. If there is non to share this is totally fine too.

**Reply by squidfunk** [reply]
No, sadly, she deleted everything.

**Reply by md42** [reply]
Thanks for the confirmation. Still, what a piece of Art <3
